### PR TITLE
enable note view for userbot

### PR DESCRIPTION
**Pull request by architdate**

### CODE REVIEW

**Comment by MrYacha** [comment]
I'm against. We already have a filters.

File: userbot/modules/notes.py [hunk 1]
```diff
@@ -64,7 +64,7 @@
         )
 
 
-@register(incoming=True, pattern=r"#\w*", disable_edited=True)
```
Comment (MrYacha) [inline userbot/modules/notes.py]:
```suggestion
@register(pattern=r"#\w*", disable_edited=True)
```
Actually we don't need incoming=True and outgoing=True together.

File: userbot/modules/notes.py [hunk 2]
```diff
@@ -64,7 +64,7 @@
...
+@register(incoming=True, outgoing=True, pattern=r"#\w*", disable_edited=True)
```
Comment (MrYacha) [inline userbot/modules/notes.py]:
```suggestion
@register(pattern=r"#\w*", disable_edited=True)
```
BTW, We don't need incoming=True and outgoing=True together, if we remove all of them it still will works well

File: userbot/modules/notes.py (hunk 1 above)
Comment (architdate) [reply userbot/modules/notes.py]: Ah, sorry, I didnt know how the defaults worked if there were no args (literally just git cloned 15 mins ago). Will modify my fork!

**Comment by architdate** [comment]
> I'm against. We already have a filters.

From what my friend read in the support chat, you wanted notes support in saved messages? This is the only way to do that I believe?

File: userbot/modules/notes.py (hunk 2 above)
Comment (architdate) [reply userbot/modules/notes.py]: Ah i just recently found out this repo, so I didnt check the default settings without args being passed to the register decorator. I'll modify my fork to reflect these changes.

**Comment by MrYacha** [comment]
I just tested with this commits, all looking good. I do not mind adding now.

**Pull request merged**